detail/namespace.vue

```diff
@@ -129,6 +164,18 @@ export default {
     statesByType() {
       return getStatesByType();
     },
+
+    /**
+     * Workload table data for current namespace
+     */
+    workloadRows() {
+      const params = this.$route.params;
+      const { id } = params;
+      const rows = getRows(this.workloads, !this.allTypes);
```

Comment:
What is `allTypes` intended to be? Looks like this is always falsey so the `showAsWorkload` filter is never applied, then e.g. a namespace with one deployment also lists the associated replicaset:
![Screen Shot 2022-03-02 at 1 22 45 PM](https://user-images.githubusercontent.com/42977925/156443458-6c1fa1b2-8786-4382-b3ef-21a37f6a2596.png)

Do we want to show spawned workloads (replicasets from deployments, jobs from cornjobs) in this view @nwmac ? The workload list view filters these out.

Reply:
Removed `allTypes` as it's not necessary.

Reply:
For some reasons it seems to do not be recognized while using Lodash :thinking: